authorization-server/package.json

```diff
@@ -41,12 +46,14 @@
     "class-transformer": "^0.5.0",
     "class-validator": "^0.13.0",
     "cookie-parser": "^1.0.0",
+    "didkit-wasm-node": "^0.1.6",
```

Comment:
The `didkit-wasm-node` is compatible only for node code and doesn't have web / browser support. To support a web service we would have to use `didkit-wasm` (functionalities and naming convention remains same). @JGiter Am I right?

Reply:
But code in this service is 100% server-side. No part of it is to be executed within the browser. If I understand you correctly :)/

Reply:
Wonderful! I wasn't aware of that, thanks.